dbt/adapters/bigquery/connections.py

```diff
@@ -555,6 +555,9 @@ def table_ref(database, schema, table_name):
     def get_bq_table(self, database, schema, identifier):
         """Get a bigquery table for a schema/model."""
         conn = self.get_thread_connection()
+        # backwards compatibility: fill in with defaults if not specified
+        database = database or conn.credentials.database
+        schema = schema or conn.credentials.schema
```

Comment:
To discuss: Backporting this piece to `1.1.latest`, to resolve https://github.com/dbt-labs/dbt-bigquery/issues/180. Let's just add a test case for the regression case as well. It's really just calling `adapter.get_columns_in_relation` on a `Relation` that's missing `database`/`schema`

Reply:
Do we want this test in the test_bigquery_adapter section using the magic mock?

Reply:
or does this fall under more of a integration test?